app/views/finders/show.html.erb

```diff
@@ -16,86 +16,32 @@
 
 <% content_for :meta_title, finder.name %>
 
-<% if finder.show_phase_banner? %>
-  <%= render 'govuk_publishing_components/components/phase_banner', phase: finder.phase, message: finder.phase_message%>
-<% end %>
-
-<% if @breadcrumbs.breadcrumbs %>
-  <%= render 'govuk_publishing_components/components/breadcrumbs', breadcrumbs: @breadcrumbs.breadcrumbs %>
-<% else %>
-  <%= render 'govuk_publishing_components/components/contextual_breadcrumbs', content_item: @raw_content_item, prioritise_taxon_breadcrumbs: finder.eu_exit_finder?  %>
-<% end %>
-
-<% if finder.government? %>
-  <%= render 'govuk_publishing_components/components/government_navigation', active: finder.government_content_section %>
+<% if finder.topic_finder? && !finder.all_content_finder? %>
+  <% content_for :body_classes, "full-width" %>
+  <% inverse = true %>
```

Comment:
If possible can we move this variable into the finder_presenter? It's a bit unclear where this was defined. Alternatively you could reuse a method with `inverse: finder.i_am_a_topic_page_finder` where it's used :)